app/components/layer-footer/template.hbs

```diff
@@ -2,10 +2,10 @@
     <div class="clearfix">
         <div class="row">
             {{#if editMode}}
-                <div  class="col-xs-3"></div>
+                <div class="col-xs-3" style="width:{{percentage}}%;"></div>
```

Comment:
Lets see if we can get rid of the 'setting style is unsafe' warngin by scheduling the setting of this percentage in the component during `willRender()` to happen in the event loop's `render` phase. https://guides.emberjs.com/v1.10.0/understanding-ember/run-loop/

Reply:
this will be a future PR as there are a lot of places that have the style being set